core/server/data/migrations/versions/3.0/01-add-stripe-customers-subscriptions-table.js

```diff
@@ -0,0 +1,34 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema/commands');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options){
+        const conn = options.transacting || options.connection;
+        const hasTable = await conn.schema.hasTable('stripe_customers_subscriptions');
```

Comment:
Should this be under the `members_` namespace too as `members_stripe_customers_subscriptions`? That's a bit of a long name, it's just the `stripe_` naming stood out as being different to anything we currently have.

Reply:
Hmm maybe?

The thinking was that we have a `members_stripe_customers` table which has `stripe_customers` belonging to `members`

This table conatins `stripe_subscriptions` belonging to `stripe_customers` so initially I thought the table should be called `stripe_customers_stripe_subscriptions` but figured the double `stripe` was not necessary.

I think you're right though, that these tables are highly members specific - so maybe `members_stripe_customers_subscriptions` would be appropriate :thinking:

Reply:
Yeah I'd vote for `members_stripe_customers_subscriptions` as well, which though long is definitely more consistent with rest of our naming and also explicit is good 😄